tests/unit/Dashboard.spec.ts

```diff
@@ -90,8 +91,8 @@ describe('Dashboard - UI', () => {
     store.state.isCoaPending = false
     await Vue.nextTick()
 
-    expect(localVm.isAllowed('addressChange')).toBe(true)
-    expect(localVm.isAllowed('directorChange')).toBe(true)
+    expect(localVm.isAllowed(AllowableActions.ADDRESS_CHANGE)).toBe(true)
+    expect(localVm.isAllowed(AllowableActions.DIRECTOR_CHANGE)).toBe(true)
```

Comment:
Per a recent PR, I'm going to start using enums (and interfaces) in the unit tests.

This is a u-turn from my previous approach where I said unit tests should be fragile. Now this may force us to update unit tests when we update an enum or interface. This is good. The point is that the unit tests should closely mimic the running code.